edx/edx-ora2
# fix: textarea render multiple times

Putting html within textarea made xblock re-render escape html on another div.

JIRA: [EDUCATOR-5806](https://openedx.atlassian.net/browse/EDUCATOR-5806)

**What changed?**

- Added if condition for rendering different response type
- The code change is mostly a revert of `pre-tinymce` feature [BD-05](https://github.com/edx/edx-ora2/pull/1576/files?file-filters%5B%5D=.html#diff-14dfd474b604ecd99ce7fbf79b1ab1ac003307e4a99972670b632b7bee911c87)
- Additional bug fix: WYSIWYG editor couldn't render image/link properly

**Developer Checklist**


**Testing Instructions**

- Normal TextArea
- Open submitted ORA

**NOTE**: To reproduce, you need to grade the response. Try refresh a couple times if you do not see the bugs.
Before:
<img width="782" alt="Screen Shot 2021-06-01 at 1 34 19 PM" src="https://user-images.githubusercontent.com/83240113/120366838-24270f80-c2de-11eb-940e-3edfb4bea716.png">

After:
<img width="915" alt="Screen Shot 2021-06-01 at 1 35 45 PM" src="https://user-images.githubusercontent.com/83240113/120367018-5a648f00-c2de-11eb-810f-239c7a6bb60f.png">

- WYSIWYG editor
  - Open submitted ORA
  - The response is properly formatted
Before:
<img width="740" alt="Screen Shot 2021-06-01 at 1 30 07 PM" src="https://user-images.githubusercontent.com/83240113/120366527-c1ce0f00-c2dd-11eb-850b-e44f89a75d9f.png"> 

After:
<img width="865" alt="Screen Shot 2021-06-01 at 1 30 20 PM" src="https://user-images.githubusercontent.com/83240113/120366563-c8f51d00-c2dd-11eb-9868-964d0add9991.png">

[**Use this if needed**](https://studio.stage.edx.org/container/block-v1:edx+TS101+2020_T1+type@vertical+block@336977df509b4580bab132749a04a8a5)
NOTE: I'm not sure if I can write unit test for this.
 
**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @edx/masters-devs-gta
